arches/app/etl_modules/bulk_data_deletion.py

```diff
@@ -141,7 +142,10 @@ def delete_resources(self, userid, loadid, graphid, resourceids):
                 resources = Resource.objects.filter(pk__in=resourceids)
             for resource in resources.iterator(chunk_size=2000):
                 resource.delete(user=user, index=False, transaction_id=loadid)
+                deleted_count += 1
             result["success"] = True
+            result["deleted_count"] = deleted_count
+            result["message"] = _(f"Successfully deleted {deleted_count} resources")
```

Comment:
I'm pretty certain this won't work -- by the time gettext is called, the variable will have already been interpolated and so it won't match any translations. I think the pattern just below this works fine.